src/toil/fileStore.py

```diff
@@ -721,6 +721,13 @@ def readGlobalFile(self, fileStoreID, userPath=None, cache=True, mutable=None):
                                                               0.0, False)
         return localFilePath
 
+    def exportFile(self, jobStoreFileID, dstUrl):
+        while jobStoreFileID in self._pendingFileWrites:
```

Comment:
Fair enough. If it comes down to personal preference I'll keep the code as is since it makes the most sense to me